src/pixelmapFeature.js

```diff
@@ -339,13 +339,14 @@ var pixelmapFeature = function (arg) {
       m_quadFeature = m_this.layer().createFeature('quad', {
         selectionAPI: false,
         gcs: m_this.gcs(),
-        visible: m_this.visible()
+        visible: m_this.visible(undefined, true)
```

Comment:
The *user* just uses `m_this.visible(true)`.  This is just done, because we don't want to change the state of visibility based on the *level*.